src/views/Rewards/Rewards.js

```diff
@@ -120,19 +142,21 @@ export default function Rewards(props) {
     totalRewardAmountEth = ethPrice.mul(userData.totalRewards);
   }
 
-  if (!rewardWeeks && selectedWeek !== undefined) {
+  if (!currentRewardWeek && selectedWeek !== undefined) {
     setSelectedWeek(undefined);
-  } else if (selectedWeek === undefined && !!rewardWeeks) {
-    setSelectedWeek(parseFloat(rewardWeeks[rewardWeeks.length - 1].week) + 1);
+  } else if (selectedWeek === undefined && !!currentRewardWeek) {
```

Comment:
reads better with the if
```suggestion
  } else if (!!currentRewardWeek && selectedWeek === undefined) {
```